cves/CVE-2018-14394.yml

```
@@ -135,7 +128,10 @@ lifetime:
           git log --stat abc..def -- your/file.c
           (where a)
       * the changelog between the two
-  answer:
+  answer: |
+    there were a lot of refactoring of this code. None of the refactoring caused 
```

```suggestion
    There was a lot of refactoring of this code. None of the refactoring caused 
```
Grammar counts, too :)